pkg/agent/flowexporter/connections/connections.go

```diff
@@ -118,13 +128,6 @@ func (cs *ConnectionStore) addOrUpdateConn(conn *flowexporter.Connection) {
 			conn.DestinationPodName = dIface.ContainerInterfaceConfig.PodName
 			conn.DestinationPodNamespace = dIface.ContainerInterfaceConfig.PodNamespace
 		}
-		// Do not export flow records of connections whose destination is local Pod and source is remote Pod.
```

Comment:
so now that we generate 2 "copies", is there any confusion when we consume the flow records in ELK?

Reply:
Yes, that's possible because of two copies of flows. Need to work with @zyiou to look into that.

I was in two minds whether to enable flow records from the destination node or disable because of flow collector implications. Need some more time. I am ok with continuing the old format of not to export to avoid confusion and take this up in a following PR that can be included in a patch release. What do you think?

Reply:
If we don't export from the destination Node, then we will never have ingress network policy information unless the source & destination nodes are the same. However, unless I am mistaken this is already the case for service information...

That wouldn't really be a candidate for a patch release, based on the impact on record consumers and the fact that this (FlowExporter) is an Alpha feature. But we could resolve this for 0.12.

I'll leave it up to you. Let's chat offline about the impact on the consumers of flow records, and especially on our reference ELK stack.